aws/resource_aws_elasticsearch_domain.go

```diff
@@ -143,6 +144,26 @@ func resourceAwsElasticSearchDomain() *schema.Resource {
 				Default:  "1.5",
 				ForceNew: true,
 			},
+			"vpc_options": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
This implementation implies that we can have more than a single vpc_options section.
What do you think of adding a `MaxItems` option set to 1, so that we can only define a single VPC configuration?

Reply:
I agree with all reviews:bow:
And after increasing timeout, I'll run testacc and report the result. Of course, I won't add the 
 timeout change.